src/views/matching/components/QuizAlertModal.vue

```diff
@@ -0,0 +1,33 @@
+<template>
+  <div class="fixed inset-0 flex items-center justify-center bg-black/70 z-20">
+    <div
+      class="relative bg-ivory rounded-4xl py-6 px-4 w-[40vh] flex flex-col items-center shadow-lg"
+    >
+      <div class="absolute right-4 top-4 size-7 cursor-pointer">
+        <img :src="system_close" alt="닫기 아이콘" @click="$emit('close')" />
+      </div>
+      <div class="text-limegreen-900 text-xl mt-9 mb-2">{{ title }}</div>
+      <br />
+      <div
+        class="text-limegreen-700 text-sm mb-10 text-center"
+        v-html="message"
+      ></div>
+      <button
+        class="w-36 h-12 bg-limegreen-500 text-ivory rounded-lg py-2 text-lg font-bold"
+        @click="$emit('confirm')"
+      >
+        {{ buttonText }}
```

Comment:
해당 컴포넌트가 퀴즈 시작 전에만 사용되는 것으로 보이는데 이후에도 다른 곳에서 재사용하지 않는다면 button-text를 프롭스로 전달받지 않고 아래처럼 작성해도 괜찮을거 같습니다!!
```suggestion
        <p>네!</p>
```